### PR TITLE
ELK post.

### PR DESCRIPTION
**Pull request by gmihaiescu**
Please review my post about using ELK and Ntopng to monitor download requests.

### CODE REVIEW

File: _posts/george_mihaiescu/2017-01-23-How-we-use-ELK-and-Ntopng-to-track-and-visualize-data-downloads.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title:  "How we use ELK (Elasticsearch, Kibana, Logstash) and Ntopng to track and visualize data downloads"
```
Comment (Fgerthoffert) [inline _posts/george_mihaiescu/2017-01-23-How-we-use-ELK-and-Ntopng-to-track-and-visualize-data-downloads.md]: I would put a smaller title. "Using ELK and Ntopng to monitor data downloads"

File: _posts/george_mihaiescu/2017-01-23-How-we-use-ELK-and-Ntopng-to-track-and-visualize-data-downloads.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    version: small
+    title: Software Engineering Blog
+    image: header-logo-crop.png
+    icon: icon-blog
+---
+
+## Introduction
+<a href="http://www.cancercollaboratory.org/">Cancer Genome Collaboratory</a> is a cloud environment built using open-source technologies that aims to offer cancer researchers access to compute resources close to the large data sets used in their analysis. 
+
+Currently we store more than 540 TB of data and have available 2500 CPU cores deployed on top of Openstack and Ceph mainly, but we also use a number of other open-source tools to manage and monitor the infrastructure.
```
Comment (Fgerthoffert) [inline _posts/george_mihaiescu/2017-01-23-How-we-use-ELK-and-Ntopng-to-track-and-visualize-data-downloads.md]: In this sentence you mix collab current usage (data) and available cores. Should we have total storage and total CPU instead ?
"Collaboratory infrastructure uses OpenStack and Ceph at its core, to provide the research community with xx PB of storage and 2500 CPU cores, but the team also relies on a number of other open-source tools to manage and monitor the infrastructure."

File: _posts/george_mihaiescu/2017-01-23-How-we-use-ELK-and-Ntopng-to-track-and-visualize-data-downloads.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+All the servers ship their logs to the central Logstash server where they are parsed and injected into Elasticsearch, so we can easily query the download requests.
```
Comment (Fgerthoffert) [inline _posts/george_mihaiescu/2017-01-23-How-we-use-ELK-and-Ntopng-to-track-and-visualize-data-downloads.md]: send their logs (instead ship their logs)

File: _posts/george_mihaiescu/2017-01-23-How-we-use-ELK-and-Ntopng-to-track-and-visualize-data-downloads.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+<figure>
+    <img src="{{site.urlimg}}george_mihaiescu/Logstash_fields.png" />
+    <figcaption>:(</figcaption>
+</figure>
+
```
Comment (Fgerthoffert) [inline _posts/george_mihaiescu/2017-01-23-How-we-use-ELK-and-Ntopng-to-track-and-visualize-data-downloads.md:32]: Add a short paragraph/description about what Kibana is and does ?

File: _posts/george_mihaiescu/2017-01-23-How-we-use-ELK-and-Ntopng-to-track-and-visualize-data-downloads.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+Finally, we can create a Kibana visualization out of the search results in order to get a better idea on the major download users (each hit count means 1 GB download).
+
+<figure>
+    <img src="{{site.urlimg}}george_mihaiescu/Large_downloads_visualization.png" />
+    <figcaption>:(</figcaption>
+</figure>
+
+In addition to the central aggregation of the application logs in ELK, the core switch connecting the environment to the Internet sends sflow statistics to <a href="http://www.ntop.org/">Ntopng</a> for which we have a free license nicely provided by the Ntop maintainers.
+
```
Comment (Fgerthoffert) [inline _posts/george_mihaiescu/2017-01-23-How-we-use-ELK-and-Ntopng-to-track-and-visualize-data-downloads.md:48]: Add a short paragraph/description about what Ntop is and does ?

File: _posts/george_mihaiescu/2017-01-23-How-we-use-ELK-and-Ntopng-to-track-and-visualize-data-downloads.md (hunk 3 above)
Comment (Fgerthoffert) [inline _posts/george_mihaiescu/2017-01-23-How-we-use-ELK-and-Ntopng-to-track-and-visualize-data-downloads.md]: Add a short paragraph/description about what Logstash is and does ?

Comment (Fgerthoffert) [reply _posts/george_mihaiescu/2017-01-23-How-we-use-ELK-and-Ntopng-to-track-and-visualize-data-downloads.md]: Add a short paragraph/description about what ElasticSearch is and does ?

File: _posts/george_mihaiescu/2017-01-23-How-we-use-ELK-and-Ntopng-to-track-and-visualize-data-downloads.md (hunk 2 above)
Comment (jmimico) [reply _posts/george_mihaiescu/2017-01-23-How-we-use-ELK-and-Ntopng-to-track-and-visualize-data-downloads.md]: Nice revision

File: _posts/george_mihaiescu/2017-01-23-How-we-use-ELK-and-Ntopng-to-track-and-visualize-data-downloads.md (hunk 5 above)
Comment (jmimico) [reply _posts/george_mihaiescu/2017-01-23-How-we-use-ELK-and-Ntopng-to-track-and-visualize-data-downloads.md:48]: Maybe replace 'nicely' with 'kindly'.

File: _posts/george_mihaiescu/2017-01-23-How-we-use-ELK-and-Ntopng-to-track-and-visualize-data-downloads.md (hunk 4 above)
Comment (gmihaiescu) [reply _posts/george_mihaiescu/2017-01-23-How-we-use-ELK-and-Ntopng-to-track-and-visualize-data-downloads.md:32]: I linked to the project's page that describes best what Kibana is.

**Review by Fgerthoffert — APPROVED** [review]
Thanks

**Pull request merged**